extensions/2.0/Vendor/EXT_feature_metadata/schema/texture.EXT_feature_metadata.featureMapping.schema.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "Texture feature mapping",
+    "type": "object",
+    "description": "An object mapping per-texel feature IDs to property arrays in a feature collection.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "featureCollection": {
+            "allOf": [ { "$ref": "glTFid.schema.json" } ],
```

Comment:
If we switch over to using a dictionary to store feature collections, then this should be changed to a string.